sys/mips/include/pcpu.h

```diff
@@ -81,7 +107,12 @@
 #ifdef _KERNEL
 
 extern char pcpu_space[MAXCPU][PAGE_SIZE * 2];
+#ifndef __CHERI_PURE_CAPABILITY__
 #define	PCPU_ADDR(cpu)		(struct pcpu *)(pcpu_space[(cpu)])
+#else /* __CHERI_PURE_CAPABILITY__ */
+#define	PCPU_ADDR(cpu)                                                  \
+    (struct pcpu *)cheri_setbounds(pcpu_space[(cpu)], sizeof(struct pcpu))
```

Comment:
Can this use `cheri_setboundsexact`?

Reply:
I think so, the fixed pcpu space is below page size.